### DISCUSSION TITLE
test_lottery_unit.py test question

### DISCUSSION THREAD

**Discussion by luahana** [smartcontractkit/full-blockchain-solidity-course-py#738] [Q&A] [answered]
Hi,

I am just wondering when you do the test_get_entrance_fee, how do you know the set etc price?
eg. where does it show the eth = 2000usd?

**Comment by luahana** [comment, marked as answer]
oh I think I found the answer.

we set the INITIAL_VALUE in helpful_script

**Reply by cromewar** [reply]
Hello @luahana yes we do set it on helpful script, I'm you solved so quickly.
